src/test/fuzz/crypto.cpp

```diff
@@ -58,6 +60,14 @@ void test_one_input(const std::vector<uint8_t>& buffer)
             (void)Hash160(data);
             (void)Hash160(data.begin(), data.end());
             (void)sha512.Size();
+
+            // MuHash3072 only accepts a fixed key length of 32 bytes
+            data.resize(32);
```

Comment:
To not interfere with the other fuzzers by changing the data size, what about either creating a new `case 3` specifically for this MuHash3072 code (the resize and the lines immediately below), or alternatively do something along the lines of `std::vector<uint8_t> muhash_data = data;` + `muhash_data.resize(32);`?

Reply:
done, opted for your second suggestion.